src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -169,8 +169,9 @@ function AuthScreens({session, lastOpenedPublicRoomID, isUsingMemoryOnlyKeys = f
         const shouldGetAllData = !!isUsingMemoryOnlyKeys || SessionUtils.didUserLogInDuringSession();
         // Sign out the current user if we're transitioning with a different user
         const isTransitioning = currentUrl.includes(ROUTES.TRANSITION_BETWEEN_APPS);
+        const isSupportalTransition = currentUrl.includes('authTokenType=support');
```

Comment:
I understand where you're coming from. Can we treat this in another PR? This is a PR that is quite long and I've added more changes than I was initially supposed. If possible I want to avoid changing the way we're doing things and that could potentially increase the risk of bugs.

Reply:
> Can we treat this in another PR?

For sure, yes. Just was writing down this thought to try and increase visibility on this weird gotcha